modules/nf-core/kmcp/search/main.nf

```diff
@@ -40,7 +40,8 @@ process KMCP_SEARCH {
     def args = task.ext.args ?: ''
     prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.gz
+    touch ${prefix}
+    gzip ${prefix}
```

Comment:
This is how we have generally been doing this
```suggestion
   echo "" | gzip > ${prefix}.gz
```

Reply:
You know what I knew there was something like that but couldn't for the life of me remember it

And I remembered that exactly this morning when I was working on another module with @aidaanva 🤣 thank you!